collections/_posts/2022-11-10-fabric.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: Fabric: A New JSON Library
+category: governance
+
+meta:
+nav: blog
+author: Matt Hicks
+---
+
+I know what you're thinking! "A new JSON library? Why? Don't we have plenty of those?" Well, the short answer is a
+resounding yes, but this library was born of the frustrations felt using those existing libraries. Don't get me wrong,
+there are some great JSON libraries in Scala and it's not my purpose to disparage them.  However, I assume others found
+the same frustrations I did with the existing libraries. This library benefited a great deal from the things I liked
+about the existing libraries and I went to great pains to make sure it didn't suffer from the same downsides they did.
```

Comment:
Hmm.  While I don't think it's the end of the world, I would simply avoid the mild negative spin here.  I'd recommend instead just focusing on what Fabric excels at.  Which, yes, will somewhat duplicate the repo's README, but that's fine -- this is a push, and it's okay to say some of the same things that are in the fuller "pull" of reading the repo.

Reply:
@jducoeur I can appreciate that, but I'm not really sure to explain what Fabric excels at without contrasting it against the existing solutions. After all, if I don't have specific reasons not to use Circe or uPickle, why would you consider using my library?

Reply:
@jducoeur I drastically simplified the text. Hopefully, it's lost any negativity.

Reply:
Yep, the changed text works for me.  Thanks!